manage.py

```diff
@@ -9,8 +7,55 @@
 migrate = Migrate(app, db)
 
 manager = Manager(app)
-manager.add_command('db', MigrateCommand)
 
-if __name__ == '__main__':
-    manager.run()
 
+@manager.command
+def init():
+    "Initiates the application for first time use."
+
+    from cleansweep.loaddata import init
+    init()
+
+
+@manager.command
+def load(directory):
+    "Loads data from the specified directory."
+
+    from cleansweep.loaddata import main
+    main(directory)
+
+
+@manager.command
+def loadfile(filename):
+    "Loads data from the specified file."
+
+    from cleansweep.loaddata import main_loadfiles
+    main_loadfiles(filename)
```

Comment:
Whoops. Will fix that. I'm supposed to create a new method for loading a single file, right?